lib/images.sh

```diff
@@ -3,8 +3,8 @@
 # Image url and checksum
 IMAGE_OS=${IMAGE_OS:-Centos}
 if [[ "${IMAGE_OS}" == "Ubuntu" ]]; then
-  export IMAGE_NAME=${IMAGE_NAME:-focal-server-cloudimg-amd64.img}
-  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://cloud-images.ubuntu.com/focal/current}
+  export IMAGE_NAME=${IMAGE_NAME:-UBUNTU_NODE_IMAGE_K8S_v1.18.8.qcow2}
```

Comment:
Could you maybe update the image to change thename ? We know that it is k8s v1.18.8 since the folder is named that way. but which Ubuntu version is it ? that might also be important to know.

Reply:
Yeah that can be added. I ll do that in a follow up PR if thats ok